### PR TITLE
chore(compiler): refactor emoji rendering logic in compiler

### PR DESCRIPTION
**Pull request by anshumanv**
refactor emoji rendering

**What kind of change does this PR introduce?**
refactor emoji rendering logic, save loops.

**Did you add tests for your changes?**
No

**If relevant, did you update the documentation?**
NA

**Summary**
Compare one whether to render emojis or not

**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

File: lib/utils/compiler.js [hunk 1]
```diff
@@ -3,21 +3,16 @@ const fs = require('fs');
...
 
 function generateOutput(outputOptions, stats, statsErrors) {
     const statsObj = stats.toJson(outputOptions);
     const { assets, entrypoints, time, builtAt, warnings, outputPath } = statsObj;
 
-    const emojies = ['✅', '🌏', '⚒️ ', '⏱ ', '📂'];
-    const visibleEmojies = emojies.map(e => showEmojiConditionally(e));
-
+    const visibleEmojies = showEmojiConditionally() ? ['✅', '🌏', '⚒️ ', '⏱ ', '📂']: new Array.fill(5).fill('');
+    console.log(visibleEmojies)
```
Comment (anikethsaha) [inline lib/utils/compiler.js]: Why this `console.log`?

File: lib/utils/compiler.js [hunk 2]
```diff
@@ -3,21 +3,16 @@ const fs = require('fs');
...
 }
 
 function generateOutput(outputOptions, stats, statsErrors) {
     const statsObj = stats.toJson(outputOptions);
     const { assets, entrypoints, time, builtAt, warnings, outputPath } = statsObj;
 
-    const emojies = ['✅', '🌏', '⚒️ ', '⏱ ', '📂'];
-    const visibleEmojies = emojies.map(e => showEmojiConditionally(e));
-
+    const visibleEmojies = showEmojiConditionally() ? ['✅', '🌏', '⚒️ ', '⏱ ', '📂']: new Array.fill(5).fill('');
```
Comment (anikethsaha) [inline lib/utils/compiler.js]:
```
new Array.fill(5).fill('');
```    

it seems that the above line should be replaced with
```
new Array(5).fill('');   //  Array() is call to constructor
```

**Comment by anshumanv** [comment]
@anikethsaha changes requested?

**Comment by anikethsaha** [comment]
> @anikethsaha what changes are you requesting? 😄

None, you have fixed those already, this is fine

**Pull request merged**